src/manifest-webapp-6-2018.json

```diff
@@ -11,10 +11,34 @@
       "src": "/android-chrome-512x512-6-2018.png",
       "sizes": "512x512",
       "type": "image/png"
+    },
+    {
+      "src": "/android-chrome-mask-512x512-6-2018.png",
+      "sizes": "512x512",
+      "type": "image/png",
+      "purpose": "maskable"
+    }
+  ],
+  "shortcuts": [
+    {
+      "name": "Vendors",
+      "url": "/vendors",
+      "description": "View Vendors wares"
+    },
+    {
+      "name": "Loadouts",
+      "url": "/loadouts",
+      "description": "Create & share Loadouts"
+    },
+    {
+      "name": "Settings",
+      "url": "/settings"
     }
```

Comment:
If you long-press an icon you can see secondary actions. Could be useful but I'm fine with removing them? Just a secondary way to quickly jump to see what Xur has, etc. Also a nice way to show off features in a secondary menu? These could also have their own custom icons. 

https://developer.mozilla.org/en-US/docs/Web/Manifest/shortcuts

Reply:
I'm into it! I didn't know this existed. Wish we could localize it.

Reply:
Localizing would be sweet. Looks like there might be a few ways to do it - not sure what that would look like app side though. 
https://stackoverflow.com/questions/57763393/localising-a-pwa-web-manifest

I think it's also mostly fairly straightforward to do this ourselves with the auto-generated apps. We can just add in the shortcuts ourselves (it's just pointing to a URL, and we handle all of the deeplinking already.) 

Might be good to just have it as-is for now though? We could maybe track hits to those endpoints from the apps to see if people are liking them? Would that be easy to do, would it help if I added a query param at the end to denote it's being hit from an app, or is that not needed?

Reply:
Would it be useful to add the other endpoints into records/progress/organizer/etc? wondering what the cut-off should be?

Reply:
Also, do we use a generic icon for "loadouts" or "vendors"? I could use the gear icon for settings.

Reply:
https://www.youtube.com/watch?v=baSiSIyTGSk&t=326s

Reply:
Ooh that helps thanks! I'll look into trying it